Move Galaxy showcase out of sandbox (#180)
elixir-europe/infectious-diseases-toolkit: showcase/covid19-galaxy.md

affiliations: [DE, European Union]
page_id: covid19-galaxy-project
related_pages:
  showcase: [korogenoest]
training:
  - name: Mutation calling, viral genome reconstruction and lineage/clade assignment from SARS-CoV-2 sequencing data
    registry: TeSS
    url: https://tess.elixir-europe.org/materials/hands-on-for-mutation-calling-viral-genome-reconstruction-and-lineage-clade-assignment-from-sars-cov-2-sequencing-data-tutorial
  - name: Automating Galaxy workflows using the command line
    registry: TeSS
    url: https://tess.elixir-europe.org/materials/hands-on-for-automating-galaxy-workflows-using-the-command-line-tutorial
---

## Introduction

The SARS-CoV-2 pandemic has generated a demand for sequencing-based viral genome analysis at an unprecedented scale.
As a result of this challenge, numerous new sequencing projects have emerged to track the pandemic on a molecular level.
Fortunately, investments made in public and open computing and storage infrastructure projects, as well as in reliable and scalable data analysis
frameworks have proven beneficial and could be employed for the required genome monitoring of SARS-CoV-2.

> Here, we demonstrate how the [Galaxy Project](https://galaxyproject.org/projects/covid19) with the help of many additional collaborators leveraged

Author: > and keeping that full link for the link of the showcase itself.

What would that look like?

> Here, we demonstrate how the [Galaxy Project](https://galaxyproject.org/projects/covid19) with the help of many additional collaborators leveraged

Reviewer: ```suggestion
Here, we describe a modular and scalable [system for FAIR analysis of SARS-CoV-2 sequencing data](https://galaxyproject.org/projects/covid19/). This effort shows how the [Galaxy Project](https://galaxyproject.org/) with the help of many additional collaborators repurposed, expanded, and combined public infrastructure and systems for storing and sharing data, and for managing data analysis workflows, to tackle a global health emergency.
system for FAIR analysis of SARS-CoV-2 sequencing data.
```

> Here, we demonstrate how the [Galaxy Project](https://galaxyproject.org/projects/covid19) with the help of many additional collaborators leveraged

Reviewer: I added a suggestion.
Would give a more direct link to the showcase, wile still making the point of the overall Galaxy Project.

---
title: An automated SARS-CoV-2 genome surveillance system built around Galaxy
contributors: [Wolfgang Maier] 
description: An automated, modular system for large-scale FAIR analysis of SARS-CoV-2 sequencing data analysis powered by the Galaxy platform.
affiliations: [DE, European Union]

> page_id: covid19-galaxy-project

Reviewer: ```suggestion
page_id: covid19_galaxy_project
```

This has the advantage that users can combine analysis modules according to their specific needs.

### Galaxy Europe

The [European Galaxy server](https://usegalaxy.eu) provides free access to powerful publicly-funded compute infrastructure and thousands of bioinformatics tools.

In the showcase we download ENA-hosted or user-requested sequencing data to the Galaxy server and process it with the workflows from WorkflowHub.

### Archive

The archive is provided by the [Centre for Genomic Regulation](https://www.crg.eu/) (CRG).

For every sample processed through Galaxy workflow runs, we are exporting key results files
(reads mapped to the viral reference sequence, variant calls, consensus sequences and tabular reports)
to a public archive so that consumers can discover, access and reuse it.

We are also maintaining a [provenance JSON file](https://galaxyproject.org/projects/covid19/data/#information-about-every-analysis-run) in this archive
that makes it possible to reconstruct the full data processing history of each sample.

You can access all the data by logging in to: `ftp://xfer13.crg.eu` as User: `FTPuser` Password: `FTPusersPassword`

### Viral Beacon project

> In the showcase the [Viral Beacon project](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations.

Reviewer: ```suggestion
In the showcase the [ CRG Viral Beacon](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations.
```
Added specific name of the tool to avoid confusion with the Beacon project in general

Reviewer: Also adding CRG Viral Beacon to the tools and resources table

  - name: Automating Galaxy workflows using the command line
    registry: TeSS
    url: https://tess.elixir-europe.org/materials/hands-on-for-automating-galaxy-workflows-using-the-command-line-tutorial
---

## Introduction

The SARS-CoV-2 pandemic has generated a demand for sequencing-based viral genome analysis at an unprecedented scale.
As a result of this challenge, numerous new sequencing projects have emerged to track the pandemic on a molecular level.
Fortunately, investments made in public and open computing and storage infrastructure projects, as well as in reliable and scalable data analysis
frameworks have proven beneficial and could be employed for the required genome monitoring of SARS-CoV-2.

Here, we describe a modular and scalable [system for FAIR analysis of SARS-CoV-2 sequencing data](https://galaxyproject.org/projects/covid19/). This effort shows how the [Galaxy Project](https://galaxyproject.org/) with the help of many additional collaborators repurposed, expanded, and combined public infrastructure and systems for storing and sharing data, and for managing data analysis workflows, to tackle a global health emergency.

> system for FAIR analysis of SARS-CoV-2 sequencing data.
> and combined public infrastructure and systems for storing and sharing data, and for managing data analysis workflows, to form a modular and scalable
> system for FAIR analysis of SARS-CoV-2 sequencing data.

Reviewer: ```suggestion
```
repeated text
